packages/react-text/src/components/Text/Text.types.ts

```diff
@@ -5,11 +5,82 @@ import { ComponentProps, ComponentState } from '@fluentui/react-utilities';
  * Text Props
  */
 export interface TextProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
-  /*
-   * TODO Add props and slots here
-   * Any slot property should be listed in the textShorthandProps array below
-   * Any property that has a default value should be listed in TextDefaultedProps as e.g. 'size' | 'icon'
+  /**
+   * Wraps the text content on white spaces.
+   *
+   * @defaultvalue true
```

Comment:
AFAIK the tsdoc annotation is pascal case https://tsdoc.org/pages/tags/defaultvalue/

```suggestion
   * @defaultValue true
```